netlib/tcp.py

```diff
@@ -529,7 +527,7 @@ def get_alpn_proto_negotiated(self):
         if OpenSSL._util.lib.Cryptography_HAS_ALPN and self.ssl_established:
             return self.connection.get_alpn_proto_negotiated()
         else:
-            return None
+            return ""
```

Comment:
`self.connection.get_alpn_proto_negotiated()` will return `""` if NO application layer protocol was negotiated.
This change will make this wrapper function to behave in the same way. Otherwise we would have two different ways to represent the state: "<no alpn performed>"